tests/test_document_clipper_pdf.py

```diff
@@ -314,6 +319,12 @@ def test_pdf_to_text_from_pdf_with_images_no_jpg(self):
         self.document_clipper_pdf_reader.pdf_to_text(self._images_to_text_method_mocked())
         self.assertEqual(1, len(self.document_clipper_pdf_reader._pdf_image_to_text_method.call_args_list))
 
+    def test_pdf_to_text_from_pdf_with_images_exception_raised(self):
+        self.pdf_file = open(PATH_TO_PDF_FILE_WITH_IMAGES)
```

Comment:
It would be nice to assert that the images directory has been emptied.

Reply:
Thats true